tests/conftest.py

```diff
@@ -111,6 +113,7 @@ def get_autoscaling():
             'max_size': 10,
             'min_size': 3,
             'instances': [{'aws_id': 'ins-1', 'ip': '192.168.20.16'}],
+            'created_time': '2018-06-06 09:59:38.127000+00:00'
```

Comment:
Does ZMON have some standard formatting? `last_modified` has format of type `"2017-08-11 11:35:15.170484"` but I'm not sure which part of ZMON infrastructure adds that to the type.

Reply:
Could be this one https://github.com/zalando-zmon/zmon-cli/blob/3caa3d97632079feb4fb4ab0c41dd25f864dcfd8/zmon_cli/cmds/entity.py#L19-L23

Reply:
@oporkka the format should probably be ISO format like `YYYY-MM-DD HH:MM:SS` which can be parsed via `time(...)`, see https://docs.zmon.io/en/latest/user/check-commands.html#time-helper-function

Reply:
Thanks, changed! Basicly it meant leaving out the timezone information `'2018-06-06 09:59:38.127000+00:00'` ->  `'2018-06-06 09:59:38.127000'`